docs/learn-proxies.md

```diff
@@ -76,6 +76,17 @@ with anonymous proxies; once you remove the proxy relationship, the account will
 
 ![anonymous proxy](assets/proxy_anonymous_diagram.png)
 
+### Time Delay Proxies
+
+We can add an aditional layer of security to proxies by giving them a delay time. The delay will be
+quantified in number of blocks (blockNumber). Polkadot and Kusama both have
+{{ block_target_in_seconds }} second blocks, hence a delay value of 10 will mean 10 blocks which
+will equal 1 minute of delay. The proxy will announce it's intended action and wait for the number
```

Comment:
Perhaps make this more clear: "for instance, a delay value of 10 indicates that there are 10 blocks, and if each block is {{ block_target_in_seconds }}, this will generate 1 minute of delay. "